.travis.yml

```diff
@@ -51,27 +51,4 @@ env:
     - PACKAGE='Three Triangulation Triangulation_2 ' 
     - PACKAGE='Triangulation_3 Union_find Visibility_2 ' 
     - PACKAGE='Voronoi_diagram_2 wininst ' 
-compiler: clang-3.6
-install: 
-  - echo "$PWD"
-  - if [ -n "$TRAVIS_PULL_REQUEST" ] && [ "$PACKAGE" != CHECK ]; then  DO_IGNORE=FALSE;  for ARG in $(echo "$PACKAGE");do if [ "$ARG" = "Maintenance" ]; then continue; fi; . $PWD/.travis/test_package.sh "$PWD" "$ARG";  echo "DO_IGNORE is $DO_IGNORE";  if [ "$DO_IGNORE" = "FALSE" ]; then  break; fi; done; if [ "$DO_IGNORE" = "TRUE" ]; then travis_terminate 0; fi;fi
-  - bash .travis/install.sh 
-  - export CXX=clang++-3.6 CC=clang-3.6;
-before_script: 
-- wget -O doxygen_exe https://cgal.geometryfactory.com/~mgimeno/doxygen_exe
-- sudo mv doxygen_exe /usr/bin/doxygen
-- sudo chmod +x /usr/bin/doxygen
-- mkdir -p build 
-- cd build 
-- cmake -DCMAKE_CXX_FLAGS="-std=c++11" -DCGAL_HEADER_ONLY=ON -DQt5_DIR="/opt/qt55/lib/cmake/Qt5" -DQt5Svg_DIR="/opt/qt55/lib/cmake/Qt5Svg" -DQt5OpenGL_DIR="/opt/qt55/lib/cmake/Qt5OpenGL" -DCMAKE_CXX_FLAGS_RELEASE=-DCGAL_NDEBUG -DWITH_examples=ON -DWITH_demos=ON -DWITH_tests=ON .. 
-- make 
-- sudo make install &>/dev/null
-- cd ..
-script: 
-- cd ./.travis
-- bash ./build_package.sh $PACKAGE
-notifications:
-  email:
-    on_success: change # default: always
-    on_failure: always # default: always
```

Comment:
That looks wrong. You have removed almost everything from the `.travis.xml`.

Reply:
Yes, the script that generates it was expecting the -3.6, i didn't see